packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -40,17 +43,29 @@ export const template = html<TableRow>`
         `)}
         <span class="row-front-spacer ${x => (x.isTopLevelParentRow ? 'top-level-parent' : '')}"></span>
         ${when(x => x.isParentRow, html<TableRow>`
-            <${buttonTag}
-                appearance="${ButtonAppearance.ghost}"
-                content-hidden
-                class="expand-collapse-button"
-                tabindex="-1"
-                @click="${(x, c) => x.onRowExpandToggle(c.event)}"
-                title="${x => (x.expanded ? tableRowCollapseLabel.getValueFor(x) : tableRowExpandLabel.getValueFor(x))}"
-                aria-hidden="true"
-            >
-                <${iconArrowExpanderRightTag} ${ref('expandIcon')} slot="start" class="expander-icon ${x => x.animationClass}"></${iconArrowExpanderRightTag}>
-            </${buttonTag}>
+            ${when(x => x.loading, html<TableRow>`
+                <span class="spinner-container">
+                    <${spinnerTag}
+                        appearance="${SpinnerAppearance.accent}"
+                        aria-label="${x => tableRowLoadingLabel.getValueFor(x)}"
+                        title="${x => tableRowLoadingLabel.getValueFor(x)}"
```

Comment:
I'm not seeing the tooltip on hover for the spinner. Is this expected?

Reply:
No, that isn't expected. I see it on the published storybook for this PR. You don't?

Reply:
Well, in Firefox, if I go to the 'loadingChildren' option for the 'Delayed Hierarchy' storybook, and hover over the spinner, I get no tooltip. In Chrome it seems to work.

Reply:
Interesting. From what I can tell, this is an existing issue in Firefox where the `title` on the spinner is very difficult to get to show up with the animation. If I force the animation to stop by applying the `--ni-private-spinner-animation-play-state:paused` CSS, then the tooltip shows up as expected.

Do you think we should do something to work around the issue?

Reply:
I don't feel strongly about it.

Reply:
I'll leave the discussion open so other can weigh in.

Reply:
It's just the title breaking not the aria-label? If so I don't feel strongly about it.

Reply:
From what I can tell, it is just the title. I think the mouse has to hover over the bits of the spinner in order for the title to be displayed, and it's pretty finicky to get that to work correctly when they keep moving.

Reply:
Oh, hmmm, that sounds less like a generic animations bug and more like something we should handle. Maybe we need an overlay div that is a pointer event target instead of the browser trying to target the moving divs. Still wouldn't hold up the PR but may be worth a tech debt issue. Definitely felt pretty clunky when I tried it

Reply:
Created #1825